src/core/map.cpp

```diff
@@ -581,7 +581,7 @@ void Map::rotateMap(double rotation, const MapCoord& center, bool adjust_georefe
 	if (adjust_declination)
```

Comment:
GitHub does have the function for comments on the commit message. Is the verb in the correct form in "... when **adjust** declination?"

Reply:
You caught me there! It should be " ... when **adjusting** declination". I left it incorrect for fear of the commit message being judged too long. For me, every commit message requires an effort at compression.

I'll fix it.

Reply:
Fixed with 4417fe7.